targets/targets.json

```diff
@@ -8,7 +8,8 @@
         "components": [
             "PSA_SRV_IMPL",
             "PSA_SRV_EMUL",
-            "NSPE"
+            "NSPE",
+            "FPGA_CI_TEST_SHIELD"
```

Comment:
@mprse adding this is intentional?

Reply:
Nice catch. 👍 

I will remove this. I added this to be able to run FPGA tests and also added this to commit by mistake.

Reply:
Fixed